src/client/pages/OfferNew/Introduction/index.tsx

```diff
@@ -97,7 +106,15 @@ export const Introduction: React.FC<Props> = ({
         </HeroImageWrapper>
         <HeroContentWrapper>
           <ContentContainer>
-            <HeroOfferDetails offerData={offerData} />
+            <HeroOfferDetailsContainer>
+              <HeroOfferDetails offerData={offerData} />
+              {hasDataCollection && (
+                <ExternalInsuranceProvider
+                  dataCollectionId={offerData.quotes[0].dataCollectionId || ''}
```

Comment:
I know that this is the old stuff back and its a fix. However, just have to ask: Is it okay to just take the first one? It just needs an Id it does not matter if its a Home or travel?

Reply:
We only have Insurely implemented in Sweden, where you always only create one quote. So in this case it should be fine.